eventmesh-storage-plugin/eventmesh-storage-rocketmq/src/main/java/org/apache/eventmesh/storage/rocketmq/producer/ProducerImpl.java

```diff
@@ -81,6 +81,13 @@ public SendResult send(CloudEvent cloudEvent) {
             messageId = sendResultRmq.getMsgId();
             sendResult.setMessageId(messageId);
             return sendResult;
+        } catch (InterruptedException e) {
+            log.error("Send message InterruptedException", e);
+            Thread.currentThread().interrupt(); // Restore interrupted status
+            throw this.checkProducerException(msg.getTopic(), messageId, e);
```

Comment:
Since you chose to re-interrupt there is no need to throw exceptions, not to mention throwing non-`InterruptedException`s.

既然您选择了重新中断就没必要再抛异常了吧，更何况抛出都是一些非`InterruptedException`。